### PR TITLE
Correct comments from #3708

### PR DESCRIPTION
**Pull request by PhilMiller**
x-ref #3708

### CODE REVIEW

File: Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H [hunk 1]
```diff
@@ -77,8 +77,8 @@ void UpdateMomentumPerezElastic (
     T_PR p1sz;
     if ( vcms > std::numeric_limits<T_PR>::min() )
     {
-        /* lorentz_tansform_factor = ( (gc-1.0)/vcms*vcDv1 - gc )*m1*g1;
-         * Rewrite to avoid roundoff problems when gc is close to 1 */
+        /* lorentz_transform_factor = ( (gc-1.0)/vcms*vcDv1 - gc )*m1*g1;
+         * Rewrite to avoid cancellation problems when gc is close to 1 */
```
Comment (dpgrote) [inline Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H]: Maybe "avoid loss of precision problems due to subtraction of similar numbers"? I don't have a strong preference - I can approve as is.

File: Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H [hunk 2]
```diff
@@ -77,8 +77,8 @@ void UpdateMomentumPerezElastic (
...
         T_PR const lorentz_tansform_factor =
             ( (gc*gc*vcms*inv_c2/(T_PR(1.0) + gc))/vcms*vcDv1 - gc )*m1*g1;
         p1sx = p1x + vcx*lorentz_tansform_factor;
```
Comment (EZoni) [inline Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H]: Shall we fix the typo `tansform` in the code variable, too?

Comment (PhilMiller) [reply Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H]: LOL, I didn't even notice that the code repeated the typo

File: Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H (hunk 1 above)
Comment (PhilMiller) [reply Source/Particles/Collision/BinaryCollision/Coulomb/UpdateMomentumPerezElastic.H]: Sure

**Review by dpgrote — APPROVED** [review]
Looks good!

**Review by EZoni — APPROVED** [review]

**Pull request merged**